webapp/sources/rudder/rudder-core/src/main/resources/Migration/dbMigration-7.1-7.2-create-campaign-table.sql

```diff
@@ -43,9 +43,10 @@ CREATE TABLE CampaignEvents (
   campaignId   text
 , eventid      text PRIMARY KEY
 , name         text
-, state        json
+, state        jsonb
```

Comment:
how will it work for systems with the rc installed? Do we need to delete the table?

Reply:
no, we "just" need a migration script that says: 
```
ALTER TABLE CampaignEvents
  ALTER COLUMN state
  SET DATA TYPE jsonb
  USING state::jsonb;
```

Reply:
ok, so we are choosing to not support update from rc (only one user with only test data).
We will need to give that user a process to migrate ("before upgrade, remove plugin, delete table `CampaignEvents`, delete plugin data directory in /var/rudder/..., do upgrade, install plugin")